src/components/MerchantsPage/DonationPool.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { SET_MODAL_VIEW } from '../../utilities/hooks/ModalPaymentContext/constants';
+import { useModalPaymentDispatch } from '../../utilities/hooks/ModalPaymentContext/context';
+import styled from 'styled-components';
+import Modal from '../Modal';
+import Hero from './images/hero.png';
+import {
+  phoneScreens,
+  tabletScreens,
+} from '../../utilities/general/responsive';
+
+const ModalBox: any = Modal;
+
+const DonationPoolBox = () => {
+  const dispatch = useModalPaymentDispatch();
+
+  const openModal = (e: any) => {
+    e.preventDefault();
+    console.log('made it here');
+    dispatch({ type: SET_MODAL_VIEW, payload: 0 });
+  };
+
+  return (
+    <React.Fragment>
+      <Container>
+        <ColumnContainer>
+          <h4>
+            Checked out our merchants but not sure who to donate to first?
+          </h4>
+          <span>You can support by donating to our donation pool!</span>
+          <span>
+            All donations will be distributed evenly to our merchants.
```

Comment:
So technically it'll only get distributed to the merchants who accept donations, which so far is all of them but might change in the future? Idk how important it is to be clear about that here. Maybe it'll just muddy up the text.

Reply:
Might just muddy up our text. I think it's fine for now, we can think about adding a disclaimer for legal reasons in the future